buildSrc/src/main/java/Dependencies.kt

```diff
@@ -0,0 +1,136 @@
+object BuildVersions {
```

Comment:
Any file in `buildSrc` folder should be placed under `src/main/kotlin` instead of `src/main/java`. Please change this.

Reply:
Done. 510fde2